lambda/MAS/appsettings.json

```
@@ -25,7 +25,7 @@
     "BucketName": "",
     "StaticURL": "",
     "CloudFront": {
-      "Enabled": "",
+      "Enabled": "false",
```

This should be an actual bool e.g. just `false`
```suggestion
      "Enabled": false,
```

---

Ah I see. Its interesting that it still parses correctly. Will octo not substitute it with a string anyway?

---

Yeah, so probably doesn't matter, just clearer to the reader that it's actually a bool. I was just going off https://github.com/nice-digital/guidance-web/blob/master/Guidance.Web/appsettings.json#L27. Good question about Octo, no idea if it knows about bools in json (I would assome it does)

---

Looks like Octo does keep bool vars as-is, see example on https://octopus.com/docs/deployment-process/configuration-features/json-configuration-variables-feature#JSONConfigurationVariablesFeature-Simplevariables